unit_tests/memory_monitor/test_memory_monitor_main.cpp

```diff
@@ -155,21 +167,23 @@ TEST_CASE ("Test mmon_add_stat", "")
   REQUIRE (test_server_info.stat_info[ret].second == allocated_size_2);
 }
 
-// Test add_stat() w/ abnormal case
+// Test add_stat() 2
 //
 // Test add_stat() with single thread when different file paths
-// that will be made same stat name are given (abnormal case)
-// There is a potential issue scenario that can't distinguish different allocation. (TODO in memory_monitor::make_stat_name ())
-// However, since such behavior is currently allowed,
-// it is necessary to test whether such behavior also returns the expected results.
+// that has multiple "/src/" in a file path.
 //
-// Expected Result: stat name "add_test.c:100" with value "allocated_size + allocated_size_2"
-TEST_CASE ("Test mmon_add_stat abnormal", "")
+// Expected Result: stat name "add_test.c:100" with value "allocated_size + allocated_size_3" and
+//                  stat name "something/thirdparty/src/add_stat.c:100" with value "allocated_size_2"
```

Comment:
I think it would be good to describe what `allocate_size, allocate_size_3, allocate_size + allocate_size_3,..` are for. 
Because, without seeing the code, it's hard to know what this case is.
e.g) 
```suggestion
// Expected Result: 
//              1) stat name "add_test.c:100" with value "allocated_size + allocated_size_3", which means ...
//              2) stat name "something/thirdparty/src/add_stat.c:100" with value "allocated_size_2", which means ..
```

Reply:
I add it.